dracut/30ignition/ignition-files.service

```diff
@@ -27,4 +30,6 @@ Before=initrd-parse-etc.service
 Type=oneshot
 RemainAfterExit=yes
 EnvironmentFile=/run/ignition.env
+# Flatcar: Make sure that the OEM mount point is there even if it shortly was away
+ExecStartPre=-systemctl start sysroot-usr-share-oem.mount
```

Comment:
Not if `ignition-mount.service` doesn't mount it again after the `umount` pre-start workaround because it doesn't have it defined in the config. The `RequiresMountsFor` gets evaluated in parallel will the other services starting up, so the mount unit is present and fulfills the requirement but before this service here is actually started it may be gone.

Reply:
Blows my mind. :)

I suppose it's something that can't be made simpler because of backward-compatibility.

Reply:
We could solve it with more downstream patches in Ignition but that's not the direction I wanted to take :upside_down_face: